testing/e2e.sh

```diff
@@ -522,6 +522,12 @@ function test_container_pull_cache() {
   rm -rf $scratch_dir
 }
 
+function test_dockerfile_image_basic() {
```

Comment:
Add todo to remove from here and move to buildkite once docker is supported

Reply:
Done